packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -195,6 +196,15 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
         ...useFormMethods,
     });
 
+    // sub-hook
+    const utxoSelection = useUtxoSelection({
```

Comment:
I don't think there is a need to re-export it in here if it's only used in `CoinControl`

Reply:
maybe it's not necessary, but I would leave it as it is if you are not strongly against